src/content/docs/en/reference/integrations-reference.mdx

```diff
@@ -196,12 +196,24 @@ A callback function to inject routes into an Astro project. Injected routes can
 - `pattern` - where the route should be output in the browser, for example `/foo/bar`. A `pattern` can use Astro's filepath syntax for denoting dynamic routes, for example `/foo/[bar]` or `/foo/[...bar]`. Note that a file extension is **not** needed in the `pattern`.
 - `entryPoint` - a bare module specifier pointing towards the `.astro` page or `.js`/`.ts` route handler that handles the route denoted in the `pattern`.
 
-Example usage:
+##### Example usage
 
 ```js
 injectRoute({
+  // Use Astro’s syntax for dynamic route syntax.
   pattern: '/foo/[dynamic]',
-  entryPoint: 'foo/dynamic-page.astro'
+  // Use relative path syntax for a local route.
+  entryPoint: './src/foo/dynamic-page.astro'
```

Comment:
I can't make a suggestion over a deleted line, but since we're not using `foo` actively anymore, can we use `/folder/` or anything else here in these two lines?